build-system/server/app.js

```diff
@@ -1181,13 +1222,51 @@ app.use('/subscriptions/skumap', (req, res) => {
   });
 });
 
+// Simulate a publisher's pingback API.
+// (amp-subscriptions)
 app.use('/subscription/pingback', (req, res) => {
   cors.assertCors(req, res, ['POST']);
   res.json({
     done: true,
   });
 });
 
+// Simulate a publisher's account registration API.
+// (amp-subscriptions-google)
+app.use('/subscription/register', (req, res) => {
```

Comment:
@ed-bird Here's a sample publisher registration API. The new `meteringRegistrationUrl` service config param points to this API

Reply:
Nice, and so they should expect to receive:
```
{
  googleSignInDetails: {
    id: 
    ...
  },
  ampReaderId: ...
}
```

Where `googleSignInDetails` object contains the fields listed here: https://developers.google.com/identity/sign-in/web/reference#googleusergetbasicprofile

Is that right?

Reply:
Great question, I added a comment to the code which hopefully answers it

---

Simulate a publisher's account registration API.

  The `amp-subscriptions-google` extension sends this API a POST request.
  The request body looks like:

  {
    "googleSignInDetails": {
      // This signed JWT contains information from Google Sign-In
      "idToken": "...JWT from Google Sign-In...",
      // Some useful fields from the `idToken`, pre-parsed for convenience
      "name": "Jane Smith",
      "givenName": "Jane",
      "familyName": "Smith",
      "imageUrl": "https://imageurl",
      "email": "janesmith@example.com"
    },
    // Associate this ID with the registration. Use it to look up metering state
    // for future entitlements requests
    // https://amp.dev/documentation/components/amp-access/#amp-reader-id
    "ampReaderId": "amp-s0m31d3nt1f13r"
  }